pkl-cli/src/test/kotlin/org/pkl/cli/CliEvaluatorTest.kt

```diff
@@ -1313,7 +1313,7 @@ result = someLib.x
     val options =
       CliEvaluatorOptions(
         CliBaseOptions(
-          sourceModules = listOf(URI("package://localhost:1/birds@0.5.0#/catalog/Ostritch.pkl")),
+          sourceModules = listOf(URI("package://localhost:1/birds@0.5.0#/catalog/Ostrich.pkl")),
```

Comment:
I understand this is a test file, and that this file imagines being versioned but I don't see any reason not to fix the spelling.

Reply:
Totally fine to change it. You'll need to update our tests, though. Take a look through the failures here to know what replacements you need to make:

https://app.circleci.com/pipelines/github/apple/pkl/1716/workflows/d5cfc639-82e2-49b6-b32a-a3d044090207/jobs/9850/tests